pybamm/models/submodels/electrode/ohm/li_metal.py

```diff
@@ -5,7 +5,30 @@
 from .base_ohm import BaseModel
 
 
-class LithiumMetalSurfaceForm(BaseModel):
+class LithiumMetalBaseModel(BaseModel):
+    def __init__(self, param, options=None):
+        super().__init__(param, "Negative", options=options)
+
+    def _get_li_metal_interface_variables(self, delta_phi_s, phi_s, phi_e):
+        param = self.param
+        pot_scale = param.potential_scale
+        delta_phi_s_dim = pot_scale * delta_phi_s
+
+        variables = {
+            "Negative electrode potential drop": delta_phi_s,
```

Comment:
Why do we have `negative electrode` variables in the lithium metal model?

Reply:
Potential drop across the li metal anode (linear)